### PR TITLE
Billing & Subscription documentation

### PR DESCRIPTION
**Pull request by elliotmjackson**
Adding documentation for billing and subscription details... files added include

1. index - `/subscription/overview`
2. manage - `/subscription/manage`
3. migration - `/subscription/migration`
4. FAQ - `/subscription/faq`

### CODE REVIEW

File: docs/bsr/faq.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+### 1. Why do I need Buf Schema Registry?
+
+   Buf Schema Registry allows you to manage and share your protocol buffer schemas with ease, making it easier for teams
+   to work together and share data across different systems and applications.
+
+### 1. What are the different pricing tiers available?
+
+   Buf Schema Registry offers multiple pricing tiers based on the number of schemas and the level of support you need.
+   The tiers range from a free plan for small projects, to higher tiers for larger projects and enterprise-level
```
Comment (maryelizbeth) [inline docs/bsr/faq.md:24]:
```suggestion
   The tiers range from a free plan for small projects to higher tiers for larger projects and enterprise-level
```

File: docs/bsr/faq.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   support.
+
+### 1. Can I try Buf Schema Registry before committing to a paid plan?
+
+   Yes, Buf Schema Registry offers a free plan that you can use to test out the product and see if it meets your needs.
```
Comment (maryelizbeth) [inline docs/bsr/faq.md:29]:
```suggestion
   Yes, Buf Schema Registry offers a free plan that you can use to test out the product to see if it meets your needs.
```

File: docs/bsr/faq.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### 1. Can I import existing protocol buffer schemas into Buf Schema Registry?
+
+   Yes, Buf Schema Registry allows you to import existing protocol buffer schemas into the system, making it easy to get
+   started and begin managing your schemas.
+
+### 1. How secure is my data in Buf Schema Registry?
+
+   Buf Schema Registry takes data security seriously and uses industry-standard security protocols to keep your data
```
Comment (maryelizbeth) [inline docs/bsr/faq.md:38]:
```suggestion
   Buf Schema Registry  uses industry-standard security protocols to keep your data
```

not an essential change, but the first bit creates too much room for interpretation of what _seriously_ means. tightening the answer gives users clearer guidance about what we are (and are not) doing

File: docs/bsr/faq.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+### 1. What kind of support is available for Buf Schema Registry?
+
+   Buf Schema Registry offers different levels of support based on your pricing tier, ranging from email support to
+   dedicated technical support and training.
+
+### 1. What integrations are available with Buf Schema Registry?
+
+   Buf Schema Registry integrates with popular tools and platforms such as GitHub, making it easy
+   to integrate with your existing development workflow.
```
Comment (maryelizbeth) [inline docs/bsr/faq.md:49]:
```suggestion
   to incorporate into your existing development workflow.
```

removes double use of integrate; incorporate may sound too stuffy - so feel free to grab a better synonym

File: docs/subscription/faq.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+hide_table_of_contents: true
+---
+
+---
+
+## Subscription tiers
+
+### What subscriptions levels are available for Buf?
+
+[Buf Subscriptions](#) come in three levels:
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:15]:
```suggestion
[Buf Subscriptions](#) come in three tiers:
```

using tiers to be consistent with industry language

File: docs/subscription/faq.md [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+
+* **Free plans** for both individuals and development teams that include unlimited public repos.
+* **Buf Team subscriptions**
+* **Buf Professional subscriptions**
+* **Buf Enterprise**
+
+### What are the differences between Free, Professional and Team Subscriptions?
+
+Professional and Team subscriptions feature additional features, support, consumption levels and more. You can compare
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:24]:
```suggestion
Professional and Team subscriptions feature additional features, support, consumption levels, and more. You can compare
```

File: docs/subscription/faq.md [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+programs at the [Buf Pricing](https://www.buf.build/pricing) page.
+
+### What are the rate limits for free subscriptions?
+
+Reasonable rate limits apply to all Buf Schema Registry traffic regardless of pricing tier.
+
+### Is it possible to enable access for our organization's IP addresses instead of tracking and managing individual accounts for our numerous agents, users, and seats?
+
+Our Enterprise package includes an Isolated Single-Tenant instance, which enables large organizations to use Buf Schema
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:33]:
```suggestion
Our Enterprise package includes an isolated single-tenant instance, which enables large organizations to use Buf Schema
```

File: docs/subscription/faq.md [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+of types in the following month, providing you with the current months grace.
+
+### How do I upgrade to a Professional plan?
+
+Upgrading your plan to a Professional plan offers you unlimited public and private repos. For information on how to
+upgrade to a Professional plan from a legacy plan, see the [How To Upgrade](manage.md#upgrade) page.
+
+### How do I upgrade to a Team plan?
+
+Upgrading your legacy plan to a Team plan offers you unlimited private repos.For information on how to upgrade to a Team
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:81]:
```suggestion
Upgrading your legacy plan to a Team plan offers you unlimited private repos. For information on how to upgrade to a Team plan,
```

File: docs/subscription/faq.md [hunk 9]
```diff
@@ -0,0 +1,218 @@
...
+### How do I downgrade from a Team plan to a Free Team plan?
+
+Before you downgrade to a Free plan, ensure that your organization details are updated to reflect features available in
+the Free plan. For example, you may need convert any private repositories to public repositories. For information on
+what’s included in the Free plan, see the [billing](index.md) page.
+
+### How do I downgrade from Professional to a Free plan?
+
+Before you downgrade to a Free plan, ensure that your account organization details are updated to reflect features
+available in the Free plan. For example, you may need to convert any private repositories to public repositories. For
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:101]:
```suggestion
available in the Free plan. For example, you may need to convert existing private repositories to public repositories. For
```

File: docs/subscription/faq.md [hunk 10]
```diff
@@ -0,0 +1,218 @@
...
+
+For information on how to add a member to a team, see [Add a member to a team](#).
+
+---
+
+## Billing questions
+
+### How can I get an invoice to purchase a Buf Team subscription?
+
+Enterprise accounts can contact the Buf team for a proposal and invoice. For Team & Professional accounts, purchasing
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:114]: Would we want to add a contact link here?

File: docs/subscription/faq.md [hunk 11]
```diff
@@ -0,0 +1,218 @@
...
+via credit card is the only option.
+
+### How do I get a receipt for my subscription?
+
+Customers will receive an email invoice each monthly billing period. A subscription confirmation with a link to
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:119]:
```suggestion
Customers will receive an email invoice each monthly billing period and a subscription confirmation with a link to
```

File: docs/subscription/faq.md [hunk 12]
```diff
@@ -0,0 +1,218 @@
...
+via credit card is the only option.
+
+### I have more questions regarding the new pricing, who can I reach out to?
+
+If you have any questions that are not covered in the FAQ, please reach out and a member of the Buf team will get back
+to you.
+
+### How does Buf Professional Subscription pricing work?
+
+Team starts at $100 USD per month for the first 100 types and each bucket of 100 types thereafter
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:134]:
```suggestion
Professional starts at $100 USD per month for the first 100 types and each increment of 100 types thereafter
```

File: docs/subscription/faq.md [hunk 13]
```diff
@@ -0,0 +1,218 @@
...
+
+### How does Buf Team Subscription pricing work?
+
+Team starts at $50 USD per month for the first 100 types and each bucket of 100 types thereafter.
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:138]:
```suggestion
Team starts at $50 USD per month for the first 100 types and each increment of 100 types thereafter.
```

File: docs/subscription/faq.md [hunk 14]
```diff
@@ -0,0 +1,218 @@
...
+- Visa
+
+### What happens if my payment fails?
+
+If your payment fails, there is a grace period of x days, including the due date. Buf retries to
+collect the payment x times using the following schedule:
+
+- ...
+
+Buf also sends an email notification `Action Required - Credit Card Payment Failed` with an attached unpaid invoice
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:165]:
```suggestion
Buf also sends an email notification with the subject `Action Required - Credit Card Payment Failed`, with an attached unpaid invoice
```

File: docs/subscription/faq.md [hunk 15]
```diff
@@ -0,0 +1,218 @@
...
+after each failed payment attempt.
+
+Once the grace period is over and the invoice is still not paid, the plan is downgraded to a free plan and all
+paid features are disabled.
+
+### What billing-related emails will I receive from Buf?
+
+Buf sends the following billing-related emails:
+
+- Confirmation of paid invoices.
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:175]:
```suggestion
- Confirmation of paid invoices.
- Notification of payment failures
- Usage overage alerts
```

File: docs/subscription/faq.md [hunk 16]
```diff
@@ -0,0 +1,218 @@
...
+### Will I receive a prorated refund if I downgrade my pricing tier?
+
+No, Buf Schema Registry does not offer prorated refunds for downgrading your pricing tier. Your new pricing tier will
+take effect at the beginning of your next billing cycle.
+
+### What happens if I exceed the number of types allowed in my pricing tier?
+
+Types are billed in increments of 100, and the cost per Type depends on the chosen plan. Usage is calculated as the
+highest count of Types that existed for an organization over the billing cycle. Buf will send an email alert to
+organization administrators when usage has progressed beyond the previously billed 100-Type bucket. Additionally, you
```
Comment (maryelizbeth) [inline docs/subscription/faq.md:199]:
```suggestion
organization administrators when usage has progressed beyond the previously billed 100-Type increment, Additionally, you
```

File: docs/subscription/index.md [hunk 17]
```diff
@@ -0,0 +1,119 @@
+---
+id: overview
+description: Buf subscription overview
+keywords: [Buf, pricing, billing, Professional, Team, business, subscription, tier, plan]
+title: Subscription Overview
+---
+
+We have recently announced updates and extensions to our product subscriptions, to enhance productivity, collaboration,
+and security for developers and businesses. As a result, Buf subscription tiers have been expanded to include Community,
+Professional, Team, and Enterprise.
```
Comment (maryelizbeth) [inline docs/subscription/index.md:10]:
```suggestion
Team, Professional, and Enterprise.
```

swap team and pro in order to preserve price order

File: docs/subscription/index.md [hunk 18]
```diff
@@ -0,0 +1,119 @@
...
+
+Each tier is outlined in detail below, and a comparison of the available features in each tier can be found in the Buf
+Pricing section.
+
+While Buf offers a free version of the Buf Schema Registry, we want to provide even greater development improvements to
```
Comment (maryelizbeth) [inline docs/subscription/index.md:15]:
```suggestion
While Buf offers a free version of the Buf Schema Registry, we want to provide an improved development experience to
```

File: docs/subscription/index.md [hunk 19]
```diff
@@ -0,0 +1,119 @@
...
+### Does Buf offer overage protection?
+
+To help protect our customers from unexpected overages, Buf offers the option for billing admins to receive an email
+alert whenever the average usage for their organization has progressed into the next tier. These email alerts are
+enabled by default and will be sent when the next tier is surpassed. Although we will **not** block uploading of more
+Types, admins will have the ability to discuss with their organization and adjust if necessary.
+
+If admins wish to receive alert emails but prefer to decrease noise, a higher threshold can be set before messages are
+sent. Assume a customer is on the Pro plan and has a desired budget of $500 USD. The alert threshold can be set at 500
+Types and no alerts will be sent until they reach that level of usage. An email alert will be sent at each level beyond
```
Comment (maryelizbeth) [inline docs/subscription/index.md:61]:
```suggestion
Types, and no alerts will be sent until they reach that level of usage. An email alert will be sent at each level beyond
```

File: docs/subscription/index.md [hunk 20]
```diff
@@ -0,0 +1,119 @@
...
+users.
+
+For a list of features available in each tier, see [Buf Pricing](https://www.buf.build/pricing/).
+
+## Buf Professional
+
+**Buf Professional** gives individual developers complete control over their development environment through a private
+instance with a dedicated subdomain, providing an integrated and dependable developer experience. This reduces the
+amount of time spent on tedious and repetitive tasks, freeing up more time for developers to create value for their
+clients.
```
Comment (maryelizbeth) [inline docs/subscription/index.md:91]:
```suggestion
organization.
```

File: docs/subscription/index.md [hunk 21]
```diff
@@ -0,0 +1,119 @@
...
+
+Buf Professional encompasses all the features of Buf Team and more, such as unlimited organizations and advanced
```
Comment (maryelizbeth) [inline docs/subscription/index.md:93]:
```suggestion
Buf Professional encompasses all the features of Buf Team as well as unlimited organizations and advanced
```

File: docs/subscription/index.md [hunk 22]
```diff
@@ -0,0 +1,119 @@
...
+collaboration and management tools. These include team and organization management with custom SSO using SAML and OIDC
```
Comment (maryelizbeth) [inline docs/subscription/index.md:94]:
```suggestion
collaboration and management tools. These include team and organization management with SSO support for SAML and OIDC
```

File: docs/subscription/index.md [hunk 23]
```diff
@@ -0,0 +1,119 @@
...
+for the entire team, comprehensive audit logging, and server admin and bulk resource management, including SCIM.
+
+For a list of features available in each tier, see [Buf Pricing](https://www.buf.build/pricing/).
+
+## Buf Enterprise
+
+**Buf Enterprise** is designed to meet the long-term governance, deployment, and service commitments of enterprises that
```
Comment (maryelizbeth) [inline docs/subscription/index.md:101]:
```suggestion
**Buf Enterprise** is designed to meet the long-term governance, deployment, and service commitments of organizations that
```

File: docs/subscription/manage.md [hunk 24]
```diff
@@ -0,0 +1,91 @@
...
+id: manage
+description: Manage your Buf subscription
+keywords: [Buf Schema Registry, upgrade, subscription, Professional, Team, business, pricing plan]
+title: Manage your subscription
+---
+
+This page contains information on how to purchase a Buf paid subscription. It also contains information on how to
+upgrade your existing subscription and update your billing information.
+
+You can upgrade from Buf Community to a Professional or Team. When
```
Comment (maryelizbeth) [inline docs/subscription/manage.md:11]:
```suggestion
You can upgrade from Buf Community to a Professional or Team plan. When
```

File: docs/subscription/manage.md [hunk 25]
```diff
@@ -0,0 +1,91 @@
...
+billing cycle after your trial ends.
+
+**5. Confirmation**
+
+Review configuration details before creating your registry and select **Create Registry**, it is typically available
+within 30 mins after subscription.
+
+Remember:
+- Types are billed in increments of 100.
+- If the average number of Types pushed to your registry exceeds 100, you will be charged at the next tier.
```
Comment (maryelizbeth) [inline docs/subscription/manage.md:46]:
```suggestion
- If the average number of Types pushed to your registry exceeds 100, you will be charged for each additional increment.
```

File: docs/subscription/manage.md [hunk 26]
```diff
@@ -0,0 +1,91 @@
...
+
+## Registry Settings Page
+
+Navigate to your organizations registry settings:
+
+1. Login to the [Buf Schema Registry](https://buf.build/login)
+2. Open the drop-down menu in the top-right corner of the page, select **Organizations**
+3. Select the Organization you manage.
+   :::note
+   You need to be the owner of the organization to manage its registry
```
Comment (maryelizbeth) [inline docs/subscription/manage.md:57]:
```suggestion
   You need to be the owner of the organization to manage its registry.
```

File: docs/subscription/manage.md [hunk 27]
```diff
@@ -0,0 +1,91 @@
...
+- If the average number of Types pushed to your registry exceeds 100, you will be charged at the next tier.
+- Minimum charge of $100/month per 100 Types after the free trial.
+
+## Registry Settings Page
+
+Navigate to your organizations registry settings:
+
+1. Login to the [Buf Schema Registry](https://buf.build/login)
+2. Open the drop-down menu in the top-right corner of the page, select **Organizations**
+3. Select the Organization you manage.
```
Comment (maryelizbeth) [inline docs/subscription/manage.md:55]:
```suggestion
3. Select the Organization you manage
```

File: docs/subscription/manage.md [hunk 28]
```diff
@@ -0,0 +1,91 @@
...
+
+Select the **Edit Payment** button, you will be taken to a checkout page where you can enter new card details.
+
+### Usage Alerts
+
+Buf will send an email alert to organization administrators when usage has progressed beyond the previously billed 100-Type
+bucket. Additionally, you will automatically have a grace period of 1 billing cycle before you are charged a higher
+amount.
+
+Admins can choose to be notified when usage has progressed into the next tier, based on average number of Types.
```
Comment (maryelizbeth) [inline docs/subscription/manage.md:87]:
```suggestion
Admins can choose to be notified when usage has progressed beyond a specified increment (e.g. 500), based on average number of Types. 
```

File: docs/subscription/migration.md [hunk 29]
```diff
@@ -0,0 +1,52 @@
...
+amount of time spent on tedious and repetitive tasks, freeing up more time for developers to create value for their
+clients.
+
+Our Professional tier gives organizations a private instance of the BSR with dedicated subdomain instead of the public 
+instance at [buf.build](https://buf.build/). When moving to your private instance, you'll need to complete a few 
+basic tasks.
+
+## Create Tokens
+
+Accessing the BSR will need to be configured to login to your new instance.
```
Comment (maryelizbeth) [inline docs/subscription/migration.md:18]:
```suggestion
In order to access the BSR, you will need to provide login configuration details via the buf CLI.
```

File: docs/subscription/migration.md [hunk 30]
```diff
@@ -0,0 +1,52 @@
...
+
+```terminal
+buf registry login example.buf.dev
+```
+
+Machine accounts will need to have tokens from the new instance generated also.
+
+## Copy Dependencies
+
+You'll want to run the following with an admin token
```
Comment (maryelizbeth) [inline docs/subscription/migration.md:28]:
```suggestion
To copy dependencies, run the following with an admin token
```

**Review by maryelizbeth — CHANGES_REQUESTED** [review]
Reviewed everything but `Types` docs for clarity, punctuation, and consistency. All suggestions optional - let me know if there are any questions.

**Review by fyockm — APPROVED** [review]

**Review by maryelizbeth — APPROVED** [review]

**Pull request merged**